### ISSUE TITLE
Need to support ops introduced or changed in opset 15 (onnx 1.10)

### ISSUE THREAD

**Issue by chinhuang007** [onnx/onnx-tensorflow#978]
class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 15, BatchNormalization);
class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 15, Bernoulli);
class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 15, Pow);
class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 15, Optional);
class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 15, OptionalHasElement);
class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 15, OptionalGetElement);
class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 15, CastLike);
class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 15, Shape);

**Comment by chinhuang007** [comment]
Quick update, the list of ops to be worked on:
BatchNormalization
Bernoulli
Pow (PR #976): done
Optional
OptionalHasElement
OptionalGetElement
CastLike
Shape (PR #977): under review

**Comment by chudegao** [comment]
BatchNormalization
Bernoulli (working)
Pow (done)
Optional (done)
OptionalHasElement (done)
OptionalGetElement (done)
CastLike  (working)
Shape (done)

if_16 (working, wait onnx pr:https://github.com/onnx/onnx/pull/3802 )
loop_16 (may not support sequence)
